interactives/models/hawk-rabbit-combo.coffee

```diff
@@ -143,8 +142,10 @@ window.model =
     env.addRule new Rule
       action: (agent) =>
         if agent.species is rabbitSpecies
-          envIndex = @getAgentEnvironmentIndex(agent)
-          brownness = @envColors[envIndex] == 'brown'
+          brownness = switch @envColors[@getAgentEnvironmentIndex(agent)]
+            when 'white' then 0
+            when 'neutral' then .5
```

Comment:
Adding a neutral scene was very easy! Except for all the images I needed to generate - definitely worth considering better solutions for the future.